### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by Jonesey13** [Fixing-Factory/FuseRatingCalculator#1] [Announcements]
Just seeing how this feature might work....

**Comment by Woolfie-T** [comment]
This is great as a POC.  I'm sure you won't be surprised to hear that I have a few comments on the detailed working!  But I'll leave that until I have time to play a bit more.
And, I'm not sure of the best place to say this as I'm not familiar with GitHub but there is a typo in the .css file, the ref to the font file has "staic" for "static."  When I did this long ago it was usual to give options for fonts in case your preferred one wasn't installed.  Is this still normal practice?  It displayed OK with a similar sans-serif for me although Poppins wasn't installed, so maybe browsers have built-in substitution tables these days.
